gtfs/spec/en/reference.md

```diff
@@ -209,244 +211,183 @@ The example below is valid, with distinct blocks every day of the week.
+|  date | Date | **Required** | Date when service exception occurs. |
+|  exception_type | Enum | **Required** | Indicates whether service is available on the date specified in the date field. Valid values are: |
+|   |  | | `1` - Service has been added for the specified date. |
+|   |  | | `2` - Service has been removed for the specified date. |
+|   |  | | *Ex. Suppose a route has one set of trips available on holidays and another set of trips available on all other days. One `service_id` could correspond to the regular service schedule and another `service_id` could correspond to the holiday schedule. For a particular holiday, the [calendar_dates.txt](#calender_datestxt) file could be used to add the holiday to the holiday `service_id` and to remove the holiday from the regular `service_id` schedule.* |
 
 ### fare_attributes.txt
 
 File: **Optional**
 
-|  Field Name | Type | Required | Details |
+|  Field Name | Type | Required | Description |
 |  ------ | ------ | ------ | ------ |
-|  fare_id | ID | **Required** | The **fare_id** field contains an ID that uniquely identifies a fare class. The **fare_id** is dataset unique. |
-|  price | Non-negative float | **Required** | The **price** field contains the fare price, in the unit specified by **currency_type**. |
-|  currency_type | Currency code | **Required** | The **currency_type** field defines the currency used to pay the fare. |
-|  payment_method | Enum | **Required** | The **payment_method** field indicates when the fare must be paid. Valid values for this field are: |
-|   |  | | * **0** - Fare is paid on board. |
-|   |  | | * **1** - Fare must be paid before boarding. |
-|  transfers | Enum | **Required** | The transfers field specifies the number of transfers permitted on this fare. Valid values for this field are: |
-|   |  | | * **0** - No transfers permitted on this fare. |
-|   |  | | * **1** - Passenger may transfer once. |
-|   |  | | * **2** - Passenger may transfer twice. |
-|   |  | | * **(empty)** - If this field is empty, unlimited transfers are permitted. |
-|  agency_id | ID | Optional | Required for feeds with multiple agencies defined in the agency.txt file. Each fare attribute must specify an agency_id value to indicate which agency the fare applies to. |
-|  transfer_duration | Non-negative integer | Optional | The **transfer_duration** field specifies the length of time in seconds before a transfer expires.  When used with a **transfers** value of 0, the **transfer_duration** field indicates how long a ticket is valid for a fare where no transfers are allowed. Unless you intend to use this field to indicate ticket validity, **transfer_duration** should be omitted or empty when **transfers** is set to 0. |
+|  fare_id | ID | **Required** | Identifies a fare class. |
```

Comment:
Thinking aloud: "Fare class" might be a somewhat misleading term, but I'm thinking we will not address it right now because we are anticipating doing future work on fares.